packages/core-utils/src/app/contract-deployment.ts

```diff
@@ -7,8 +7,17 @@ import { Provider } from 'ethers/providers'
 
 /* Internal Imports */
 import { ContractDeploymentFunction } from '../types'
+import { add0x } from '../app'
 import { sleep } from './misc'
 
+const {
```

Comment:
Totally immaterial, but Is there a reason to do this instead of 
```
import { keccak256, stripZeros, hexlify, RLP, getAddress } from 'ethers/utils'
```
?

Reply:
Ah, I didn't realize that would work! Updated 👍 -> [Import from 'ethers/utils' instead of using a deconstructor](https://github.com/ethereum-optimism/optimism-monorepo/pull/48/commits/8561cce01690f2db842254a19c7404a881316104)